docker-compose-host.yml

```diff
@@ -7,7 +7,7 @@ services:
       - credentials_node_modules:/edx/app/credentials/credentials/node_modules
       - ${DEVSTACK_WORKSPACE}/src:/edx/src:cached
   discovery:
-      volumes:
+    volumes:
```

Comment:
Need to make this repo private.

Reply:
We can't make this private since this is forked from an open source repo.

Reply:
Also, I've made sure that our custom code isn't added to the docker container and that no one can clone it unless they are given explicit access on github. So no worries there.

Reply:
:+1: